pkg/clusteragent/admission/mutate/cwsinstrumentation/cws_instrumentation.go

```diff
@@ -477,7 +487,7 @@ func (ci *CWSInstrumentation) injectCWSCommandInstrumentation(exec *corev1.PodEx
 				metrics.CWSExecInstrumentationAttempts.Observe(1, ci.mode.String(), "false", cwsPodNotInstrumentedReason)
 				return false, nil
 			}
-			cwsInstrumentationRemotePath = filepath.Join(cwsMountPath, "cws-instrumentation")
+			cwsInstrumentationRemotePath = filepath.Join(cwsMountPath, ci.directoryForRemoteCopy, "cws-instrumentation")
```

Comment:
nit: The `cwsInstrumentationRemotePath` could be done outside the `if/else` with just the `cwsMountPath` being set by the `if/else`.

Reply:
Done 👍🏻